packages/commonwealth/client/scripts/views/components/component_kit/cw_community_avatar.tsx

```diff
@@ -36,7 +36,7 @@ export const CWCommunityAvatar = (props: CommunityAvatarProps) => {
     <div
       className={getClasses<{ onClick: boolean; size: IconSize }>(
         { onClick: !!onClick, size },
-        ComponentType.CommunityAvatar
+        ComponentType.CommunityAvatar,
```

Comment:
how about this? did you fix your prettier? there should not be comma at the EOL here